components/DefaultButton.js

```diff
@@ -0,0 +1,84 @@
+import styled from "styled-components";
+import { css } from "styled-components";
+import Link from "next/link";
+import Image from "next/image";
+
+export default function DefaultButton({ url, icon, $orange, children }) {
```

Comment:
```suggestion
export default function DefaultLinkButton({ url, icon, $orange, children }) {
```

Reply:
or just default Link because its a link that just looks like a button ideally :D